applications/nrf5340_audio/src/drivers/cs47l63_comm.c

```diff
@@ -27,15 +29,16 @@ LOG_MODULE_REGISTER(CS47L63, CONFIG_CS47L63_LOG_LEVEL);
 #define CS47L63_PROCESS_THREAD_DELAY_MS 10
 
 static const struct gpio_dt_spec hw_codec_gpio =
-	GPIO_DT_SPEC_GET(DT_NODELABEL(hw_codec_gpio_in), gpios);
+	GPIO_DT_SPEC_INST_GET(0, gpio9_gpios);
```

Comment:
So now the HW codec pins are not defined in the board files?

Reply:
They are, just in a different location: https://github.com/zephyrproject-rtos/zephyr/pull/55675

Reply:
Aha, what does the name "gpio9_gpios" mean? To me it sounded like a generic GPIO define

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/4d9e13115140abb4a86dda3b4a5c7145a20949cd/dts/bindings/sound/cirrus%2Ccs47l63.yaml#L64-L67